Create README.md

JimmyTan823 opened the pull request:
wwdc21

thisIsTheFoxe reviewed (changes requested):
Hi, thanks for your contribution. Can you just move your entry up (below "Jia"), to match the *alphabetical order by first name*? :)

thisIsTheFoxe reviewed (approved):
Hi again. I just resolved the conflicts, fixed the alphabetical order and removed your LinkedIn tracking queries. I hope everything is good. If still something's missing or wrong, feel free to open another PR :)

JimmyTan823 commented:
Thank you！



发自我的iPhone


------------------ Original ------------------
From: Henry ***@***.***&gt;
Date: Wed,Apr 21,2021 2:58 PM
To: wwdc/2021 ***@***.***&gt;
Cc: JimmyTan823 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [wwdc/2021] Create README.md (#38)





 
@thisIsTheFoxe approved this pull request.
 
Hi again. I just resolved the conflicts, fixed the alphabetical order and removed your LinkedIn tracking queries. I hope everything is good. If still something's missing or wrong, feel free to open another PR :)
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
